src/App.css

```diff
@@ -332,9 +332,7 @@ Footer {
   box-sizing: border-box;
   position: relative;
   background: #D81921;
-  bottom: 3rem;
   margin-top: 0rem;
-  height: 536px;
```

Comment:
I noticed a duplicate list of footer links and a chunk of white space. I placed this in as a temporary fix till we have more time to go back.